### ISSUE TITLE
Script to update rsmcompare test files

### ISSUE THREAD

**Issue by aloukina** [EducationalTestingService/rsmtool#413]
We have experiments within RSMCompare tests that we need to update when we have changes to output files. The experiments match the existing tests for either RSMTool or RSMEval but it would be nice to have script that automatically replaces the relevant files using test_outputs.

**Comment by desilinguist** [comment]
I think this is a more general issue. For example, I just ran into this for an rsmpredict test where the SKLL model needs to be updated.

**Comment by aloukina** [comment]
For that we have a script! https://github.com/EducationalTestingService/rsmtool/blob/master/tests/update_skll_models.py

**Comment by desilinguist** [comment]
Right, I totally forgot :)

**Comment by aloukina** [comment]
The solution I adapted is: 
- Make sure that any input experiments used in RSMCompare/RSMSummarize tests are the existing test experiments for RSMTool and RSMEval and named in the same way
- Make changes to `FileUpdater` to not only check and update test output files but also check and update input files to RSMCompare/RSMSummarize if the outputs of the corresponding experiments have changed.
- I have left the RSMPredict experiments as is for now.

Drawbacks:
- If the `FileUpdater` the input files, the outputs of RSMCompare/RSMSummarize experiments will also likely change. This means that one needs to re-run the tests and re-update the outputs. I have described this process in documentation. Not sure there is anyway to bypass the two-step process if the idea is to update test input files. 
- The current approach will not work if we need to have a test which tests for compatibility of RSMSummarize/RSMCompare with outputs of the older of RSMTool.  At this point we don't have any such tests and in RSMTool 8.0 we explicitly removed any such compatibility. If we add it in future, we might consider flagging such tests as "legacy" or "do not update". It will also make it easier to deprecate them as necessary.
